### PR TITLE
Cross proxy

### PR DESCRIPTION
**Pull request by enxebre**
This provides support for proxying mesos request through node js server so we can reuse same code base while keeping both approaches either deploying this app via marathon or running it as the default mesos ui

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,5 +11,5 @@ build_ui_snapshot:
 	gulp build
 
 build: build_ui_snapshot
-	sed "s/VERSION/$(MESOS_VERSION)/g" Dockerfile > TempDockerfile
+	sed "s/VERSION/$(MESOS_VERSION)/g" DockerfileMesos > TempDockerfile
 	docker build -f TempDockerfile -t capgemini/mesos-ui:$(MESOS_UI_VERSION) .
```
Comment (wallies) [inline Makefile]: Can we build both the standalone and integrated dockerfiles in the Makefile

**Comment by wallies** [comment]
Can we update the readme as well, with how the different Dockerfiles work

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,15 @@
-FROM mesosphere/mesos-master:VERSION
-MAINTAINER alberto.garcial@hotmail.com
-COPY build/master /web-ui/master
-ENV MESOS_WEBUI_DIR /web-ui
+FROM mhart/alpine-node:4.1.2
```
Comment (wallies) [inline Dockerfile]: change to 4.2.2 as this is node-lts version

File: Makefile (hunk 1 above)
Comment (enxebre) [inline Makefile]: yupp

File: Dockerfile (hunk 2 above)
Comment (enxebre) [inline Dockerfile]: let's get this in first, think that should belong to different PR
